setup.py

```diff
@@ -0,0 +1,26 @@
+from setuptools import find_packages, setup
```

Comment:
Should we move towards the newer packaging methodology using a `pyproject.toml` file, `pip` as frontend and  `setuptools` as backend.

Reply:
I talked to Jaime about this, and he says that you could do that, but it's probably not necessary for a project of this size.